README.md

```diff
@@ -7,7 +7,16 @@ Please refer to the [mbed-os-example-sockets](https://github.com/ARMmbed/mbed-os
 
 You can build the project with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command-line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
 
-1. Install Mbed CLI.
+## Mbed OS build tools
```

Comment:
Line 8 above needs to be either updated or removed, as it links to the old tools only.

Reply:
This is also the case in https://github.com/ARMmbed/mbed-os-example-blinky/blob/development/README.md - we will also need to fix it there.

Reply:
Yep, there are just many examples to maintain. Before we find a way to combine examples or factorise docs, I guess keeping things short & simple is the best.